python/src/equistore/operations/slice.py

```diff
@@ -0,0 +1,395 @@
+import warnings
+
+import numpy as np
+
+from ..tensor import TensorMap
+from ..block import TensorBlock
+from ..labels import Labels
+
+
+def slice(tensormap: TensorMap, samples_to_slice=None, properties_to_slice=None):
+    """
+    Slices an input :py:class:`TensorMap` along the samples and/or properties
+    dimension(s). `samples_to_slice` and `properties_to_slice` are
```

Comment:
```suggestion
    dimension(s). ``samples_to_slice`` and ``properties_to_slice`` are
```

RestructuredText syntax is different from markdown unfortunately

Reply:
This, and other docstring-related changes also suggested in this comment, are fixed in ae803ab9a838accd30dc9e84c02839014996c4fc.